runners/console/index.ts

```diff
@@ -410,10 +411,58 @@ export class Console extends Runner {
         if(!fs.existsSync(workspacesDir))
             fs.mkdirSync(workspacesDir);
         this.setVariable(this.workspaceDirectory, workspacesDir);
+     
+      return result;
+    }
 
-        return result;
+    runExecuteCommand(runCommand: RunCommand): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+        let exeCommand = runCommand.command.parameters[0];
+        
+        exeCommand = (runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].args)
+        ? exeCommand+ " " +runCommand.command.parameters[1].args.join(" ")
+        : exeCommand;
+
+        let dirPath = (runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].dir)
+        ? path.join(this.getWorkingDirectory(), runCommand.command.parameters[1].dir)
+        : this.getWorkingDirectory();
+
+        if(runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].asynchronous){
+            let process = ConsoleUtils.executeCommandAsync(exeCommand, dirPath, result,this.env);
+            if(process.pid && runCommand.command.parameters[2].port) {
+                this.asyncProcesses.push({ pid: process.pid, name: "ExecuteCommand", port: runCommand.command.parameters[2].port});
```

Comment:
@denise-khuu @MarcelDiessner I think we have to think of a way to deal with this case. Otherwise we have the same problem as with the other asynchronous operations  and the buildRun script will never finish.
The port should here also be  a mandatory attribute I think. And the name is also used in the killAsyncProcesses method.  Maybe we have to remove it from there and just kill all processes on the given port

Reply:
Isn´t the port mandatory already? In the fix/serverIsReachable there is a check if the assert information is transferred and I check if a port is given.

Reply:
Maybe we can use a default port, which will be used if no port is given as a parameter

Reply:
@EduardKrieger Well in your implementation it is mandatory, but we need to always push the Process into the asyncProcess Array, so it can be killed later. Right now it is only pushed if a port is given.

Reply:
Our documentation says you have to give a port, we can check if a port is given before we start executeCommandAsync so we can throw an Error if a port is missing.